epochalypse.py

```diff
@@ -123,7 +126,10 @@ def main():
     fromEpoch(float(args.epoch_input))
     print('')
   elif args.hexadecimal_input:
-    epoch = fromHex(args.hexadecimal_input)
+    hex_text = args.hexadecimal_input.replace(' ', '')
+    if args.revhex:
+      hex_text = hex_text.decode('hex')[::-1].encode('hex')
```

Comment:
That's python2, this new version of the script is python3 and decode method on string does not work.